packages/core/src/header-builder/add-csrf-token-and-cookies.ts

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
I am not so happy with the file name, but I have also no better idea. Perhaps since there is only one method which is deprecated: `legacy-csrf-token-header.ts`.